Sources/TuistEnvKit/Updater/Updater.swift

```diff
@@ -44,10 +40,14 @@ final class Updater: Updating {
             } else {
                 logger.notice("Installing new version available \(highestRemoteVersion)")
                 try installer.install(version: highestRemoteVersion.description)
+                logger.info("Updating tuistenv", metadata: .section)
+                try envInstaller.install(version: highestRemoteVersion.description)
             }
         } else {
             logger.notice("No local versions available. Installing the latest version \(highestRemoteVersion)")
             try installer.install(version: highestRemoteVersion.description)
+            logger.info("Updating tuistenv", metadata: .section)
+            try envInstaller.install(version: highestRemoteVersion.description)
```

Comment:
I woould move this outside of the if/else, and change the inner if in a guard in case "There are no updates available"